mreg_cli/util.py

```diff
@@ -40,6 +40,19 @@ def host_exists(name: str) -> bool:
     return True
 
 
+def host_info_by_id(id: int) -> dict:
+    """
+    Return a dict with host information of the host having the given id.
+
+    :param id: id of the host in the datbase.
+    :return: A dict of the JSON object received with the host information
+    """
+    path = f"/hosts/?id={id}"
+    history.record_get(path)
+    host = get(path).json()
+    return host["results"][0]
```

Comment:
The `get` can result in a empty results list, and then the return will crash.

Reply:
This subrouting will be gone in the next commit, since we drop listing all sshfp records and no other code needs it.